app/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,21 @@
+<resources>
+    <string name="app_name" translatable="false">CrashReportToMarkdown</string>
+    <string name="action_settings">Settings</string>
+    <string name="title">The incredible NPE to Markdown converter</string>
+    <string name="paste_info">Paste the JSON bugreport here</string>
+    <string name="transform">transform</string>
+    <string name="hint_markdown">Click transform and your human readable bugreport will appear here.</string>
+    <string name="copy_to_clipboard">copy to clipboard</string>
+    <string name="empty_text">Please insert some text!</string>
+    <string name="copied_to_clipboard">Copied to clipboard!</string>
+    <string name="text_cleared">Text cleared</string>
+    <string name="already_empty">Already empty!</string>
+    <string name="wip">Work in progress</string>
+    <string name="error_occured">Cannot convert bug report to Markdown</string>
+    <string name="preview">Preview the human\nreadable bugreport</string>
```

Comment:
I think "Preview" alone is enough here

Reply:
I know. But it didn't look centered with "preview" only, and didn't want to bother me with UI, that's why I wrote a sentence xD